src/xml.tsx

```diff
@@ -124,7 +124,7 @@ export function SvgXml(props: XmlProps) {
 
 export async function fetchText(uri: string) {
   const response = await fetch(uri);
-  return await response.text();
+  return response.ok ? await response.text() : null;
```

Comment:
I think it would be good to add some message e.g. in a form of `console.warn` if the `response.ok` is `false`, so we could get information what went wrong. Then I think it is ready to be merged :tada:

Reply:
Thanks, makes sense.